core/src/main/java/io/grpc/internal/ServerCallImpl.java

```diff
@@ -90,7 +89,7 @@ public void sendHeaders(Metadata headers) {
     } else {
       if (messageAcceptEncoding != null) {
         // TODO(carl-mastrangelo): remove the string allocation.
-        if (!Iterables.contains(
+        if (!GrpcUtil.iterableContains(
             ACCEPT_ENCODING_SPLITTER.split(new String(messageAcceptEncoding, GrpcUtil.US_ASCII)),
```

Comment:
Just use the old splitToList().contains(...) ?

Reply:
`splitToList()` is marked as `@Beta` and 2b1363d58600ac49b73116e00d17302063ef532a was moving away from beta APIs.

Reply:
then `new ArrayList<>(...).contains()`

Reply:
Although the additional allocation from ArrayList is not substantially more than that what's already there, what's wrong with the helper method?